<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cinema</class>
 <widget class="QWidget" name="cinema">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonNextWeek">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Week</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollAreaSchedule">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutScheduleContainer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>419</width>
      <height>299</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCurrentWeek">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>410</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>This Week</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeekRange">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>              labelWeekRange</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
